src/pages/Beta/Swap/index.tsx

```diff
@@ -24,7 +23,7 @@ const SwapUI = () => {
       {CHAINS[chainId].mainnet && (
         <GridContainer isLimitOrders={isLimitOrders}>
           {isLimitOrders && <LimitOrderList />}
-          <MyPortfolio isLimitOrders={isLimitOrders} />
```

Comment:
as you have removed necessary files for portfolio, please remove reducers / hooks if not required now in interface